Index: plymouth/themes/spinner/spinner.plymouth.desktop
===================================================================
--- plymouth.orig/themes/spinner/spinner.plymouth.desktop
+++ plymouth/themes/spinner/spinner.plymouth.desktop
@@ -8,11 +8,11 @@ Font=Cantarell 12
 TitleFont=Cantarell Light 30
 ImageDir=@PLYMOUTH_THEME_PATH@/spinner
 DialogHorizontalAlignment=.5
-DialogVerticalAlignment=.382
+DialogVerticalAlignment=.7
 TitleHorizontalAlignment=.5
-TitleVerticalAlignment=.382
+TitleVerticalAlignment=.7
 HorizontalAlignment=.5
-VerticalAlignment=.7
+VerticalAlignment=.8
 WatermarkHorizontalAlignment=.5
 WatermarkVerticalAlignment=.96
 Transition=none
